fuzzers/072-ordered_wires/run_fuzzer.py

```diff
@@ -49,6 +56,14 @@ def main(argv):
     with Pool(processes=nbParBlock) as pool:
         pool.map(start_vivado, argList)
 
+    if modBlocks != 0:
+        print("Caculate extra block")
+        start = nbBlocks * blocksize
+        stop = pipscount
+        bID = nbBlocks
+        start_vivado((bID, start, stop))
+        nbBlocks = nbBlocks + 1
```

Comment:
Why not add this as another item in argList?

Reply:
Good point ! I'm not used to Pool handling all the logic glue itself !